spacy/lang/punctuation.py

```diff
@@ -39,8 +39,20 @@
         r"(?<=[{al}{q}])\.(?=[{au}{q}])".format(
             al=ALPHA_LOWER, au=ALPHA_UPPER, q=CONCAT_QUOTES
         ),
-        r"(?<=[{a}]),(?=[{a}])".format(a=ALPHA),
+        r"(?<=[{a}])(?:{p})(?=[{a}])".format(a=ALPHA, p=PUNCT),
```

Comment:
Why is this change included in this PR?

Reply:
Sorry, I'm not ready for this to be reviewed yet. I'll take it out of draft once I am.